siriuspy/siriuspy/devices/orbit_interlock.py

```diff
@@ -964,8 +842,12 @@ def trans_thresminy(self):
 
     @trans_thresminy.setter
     def trans_thresminy(self, value):
-        if isinstance(value, (int, float)):
+        if isinstance(value, (int, float, _np.int, _np.float)):
```

Comment:
Since there is repetition, you could move this code to a auxiliary method.

Reply:
done!